src/Date/Local.fs

```diff
@@ -29,7 +30,9 @@ module Local =
         { Months : Months
           AbbreviatedMonths : Months
           Days : DaysOfWeek
-          AbbreviatedDays : DaysOfWeek }
+          AbbreviatedDays : DaysOfWeek
+          DefaultFormat : string
+          FirstDayOfTheWeek : int }
```

Comment:
Maybe that should be `DayOfWeek` instead of an int?

Reply:
Yes, using `DayOfWeek`seems a good idea. Like that users can't enter invalid day.

Also, like that they don't need to know Sunday is id 0, etc.

Reply:
Done that too